### PR TITLE
Avatars and Usernames color change

### LINKED ISSUE #1603: New avatars placeholder colors and username coloring
https://www.figma.com/file/iIjwDl9HjaaF1vlDU96h8K/INBOX-%C2%B7-Compound-iOS%2FAndroid-Components?type=design&node-id=138-23526&mode=design&t=KAPx9fxf2GdEZLGg-0

### PR DESCRIPTION
**Pull request by Velin92**
- New color for avatars and username.
- Also the font of the placeholder is now computed proportionally to the encapsulating view.
- Placeholder in notifications is also going to be updated
- This also fixes the misalignment between web and el-x (when web will support the new colouring system ofc)

I might need to update all the UI tests.

Simulator Screenshot - iPhone 14 - 2023-08-31 at 20 06 07
Simulator Screenshot - iPhone 14 - 2023-08-31 at 20 05 48

fixes #1603

### CODE REVIEW

File: DesignKit/Sources/Colors/ElementColors.swift
Comment (pixlwave) [inline DesignKit/Sources/Colors/ElementColors.swift:1]: Almost empty now 👏

File: ElementX/Sources/Other/EffectsScene/EffectsScene.swift [hunk 1]
```diff
@@ -18,6 +18,13 @@ import SceneKit
 import SwiftUI
 
 class EffectsScene: SCNScene {
+    static let colors: [Color] = [
+        .blue,
+        .red,
+        .yellow,
+        .green
```
Comment (pixlwave) [inline ElementX/Sources/Other/EffectsScene/EffectsScene.swift:25]: I'll come back and theme these when we're not busy 😅

File: ElementX/Sources/Other/Extensions/UNNotificationContent.swift [hunk 2]
```diff
@@ -184,7 +184,8 @@ extension UNMutableNotificationContent {
     }
 
     private func getPlaceholderAvatarImageData(name: String, id: String) async -> Data? {
-        let fileName = "notification_placeholder_\(name)_\(id).png"
+        // The version value is used in case the design of the placeholder is updated to force a replacement
+        let fileName = "notification_placeholderV2_\(name)_\(id).png"
```
Comment (pixlwave) [inline ElementX/Sources/Other/Extensions/UNNotificationContent.swift:188]: Good catch, I wouldn't have thought about these.

File: project.yml [hunk 3]
```diff
@@ -51,8 +51,7 @@ packages:
     path: DesignKit
   Compound:
     url: https://github.com/vector-im/compound-ios
-    revision: 50bb7cf313bd1ad17201fc7e4c1184737a0f44c2
-    # path: ../compound-ios
+    revision: 71b0c887880de639c854600a9ada6b5bcc93c719
```
Comment (pixlwave) [inline project.yml:54]: Can we leave this in lease :)
```suggestion
    revision: 71b0c887880de639c854600a9ada6b5bcc93c719
    # path: ../compound-ios
```

**Review by pixlwave — APPROVED** [review]
These look sooo good. Thank you for taking the time to implement this 🙌

**Review by amshakal — COMMENTED** [review]
So fast!!

**Pull request merged**